tools/link.py

```diff
@@ -2071,7 +2071,8 @@ def fix_es6_import_statements(js_file):
   src = read_file(js_file)
   write_file(js_file, src
              .replace('EMSCRIPTEN$IMPORT$META', 'import.meta')
-             .replace('EMSCRIPTEN$AWAIT$IMPORT', 'await import'))
+             .replace('EMSCRIPTEN$AWAIT$IMPORT', 'await import')
+             .replace('EMSCRIPTEN$AWAIT(createWasm())', 'await creasteWasm()'))
```

Comment:
```suggestion
             .replace('EMSCRIPTEN$AWAIT(createWasm())', 'await createWasm()'))
```

Reply:
Ooops.. how did that even work.

Reply:
I see.. the only effected test that I ran doesn't actually try to execute the resulting code: other.test_codesize_minimal_esm